server/sonar-server/src/test/java/org/sonar/server/permission/ws/template/SearchTemplatesActionTest.java

```diff
@@ -175,14 +172,6 @@ public void fail_if_not_logged_in() {
   }
 
   @Test
-  public void fail_if_not_global_admin() {
```

Comment:
Why have you removed this test ? It should just be replaced by a test that check that the user needs to be logged.